modules/020-deckhouse/monitoring/prometheus-rules/alerting.yaml

```diff
@@ -34,6 +34,40 @@
       description: |
         Deckhouse release is waiting for manual approval.
 
+        Please run `kubectl patch DeckhouseRelease {{ $labels.name }} --type=merge -p='{"approved": true}'` for confirmation.
+      summary: |
+        Deckhouse release is waiting for manual approval.
+  - alert: DeckhouseReleaseIsWaitingManualApproval
+    expr: max by (name) (d8_release_waiting_manual) == 2
```

Comment:
If I understand this right, a user will receive three alerts, one for each waiting version. It would be better to group them then.

Reply:
We will have only one metric here. See the upper comment